docs/getting-started/supported-languages-and-tools.md

```diff
@@ -340,6 +340,13 @@ The table below lists all languages and frameworks that Codacy supports and the
       <td><a href="https://github.com/kucherenko/jscpd">jscpd</a></td>
       <td><a href="https://eslint.org/">ESLint</a></td>
     </tr>
+    <tr>
+      <td>Unity</td>
+      <td><a href="https://github.com/dotnet/roslyn-analyzers">Roslyn Analyzers</a><a href="#client-side"><sup>1</sup></a></td>
```

Comment:
Do you agree that the new tool name is "Roslyn Analyzers" and that the tool URL is https://github.com/dotnet/roslyn-analyzers?

Reply:
yup